src/transformers/convert_graph_to_onnx.py

```diff
@@ -0,0 +1,141 @@
+from argparse import ArgumentParser, Namespace
+from os import mkdir, listdir
+from os.path import exists, abspath, dirname
+from typing import Dict, Tuple, List
+
+from transformers import is_torch_available, is_tf_available
+from transformers.pipelines import SUPPORTED_TASKS, pipeline, Pipeline
+from transformers.tokenization_utils import BatchEncoding
+
+
+class OnnxConverterArgumentParser(ArgumentParser):
```

Comment:
Here you can maybe use `HfArgumentParser` which is dataclass-based and may be more compact.

Up to you though, this works too :)